testsuite/MDAnalysisTests/core/test_copying.py

```diff
@@ -109,7 +109,6 @@ def test_move_residue(self, refTT):
     (ta.Tempfactors, float),
     (ta.Masses, float),
     (ta.Charges, float),
-    (ta.Bfactors, float),
```

Comment:
Shouldn't this still work for now?

Reply:
It's not a test, it's a helper function to make a new class with the Bfactor topologyattribute class which no longer exists:

```python
test_copying.py:110: in <module>
    (ta.Bfactors, float),
E   AttributeError: module 'MDAnalysis.core.topologyattrs' has no attribute 'Bfactors'
```